packages/mobile/e2e/Transfer.spec.js

```diff
@@ -4,19 +4,30 @@ import { enterPin, skipTo, sleep } from './utils'
 const SAMPLE_BACKUP_KEY =
   'nose inherit merry deal scout boss siren soul piece become better unit observe another horn ranch velvet kid frog pretty powder convince identify guilt'
 const VERIFICATION_COUNTRY = 'Germany'
-const VERIFICATION_PHONE_NUMBER = '+14155555555'
+const VERIFICATION_PHONE_NUMBER = '030 901820'
```

Comment:
Is this a real number? Since we're open source we should be mindful to only use fake numbers. That's why the '555' based american numbers are helpful (those can never be real #s)

Reply:
Is's a number I made out combining a couple of example ones I found online.

Reply:
Googling it shows it's a real number for something here in Germany. We shouldn't use their number here. Any reason not to use a fake one like we used to?

Reply:
oh, nice catch.